### ISSUE TITLE
Install error: dependency `parsimonious` is not available

### ISSUE THREAD

**Issue by Pinjasaur** [etienne-monier/LedgerTools#2]
First, thanks for the package!

I was running it fine for a few weeks when it suddenly stopped working. My install is on Windows, Sublime Text Dev Build 3210.

After removing & attempting to re-install I'm able to get some basic logs from the <kbd>Ctrl + `</kbd> command prompt in Sublime.

The relevant error messages appear to be:

```
Package Control: The dependency 'parsimonious' is not currently installed; installing...
Package Control: The dependency 'parsimonious' is not available
Package Control: The dependency 'parsimonious' could not be installed or updated
```

My initial searching didn't yield any helpful debugging results so I figured I'd make a ticket here to see if you had any ideas. Thanks!

**Comment by etienne-monier** [comment]
I can't have access to my local repo for the moment (French winter holidays). What I suggest you is to remove the dependencies.json file. I thought I had suppressed it, but it's still here.
Look in the .config/sublime-text-3/Packages/LedgerTools dir.

**Comment by Pinjasaur** [comment]
Tried cloning the repo directly into the `Packages` directory and it seemed to do the trick. Would prefer to keep it updated with Package Control but I'm fine with this workaround in the meantime. Thanks!

**Comment by etienne-monier** [comment]
Ok, that was the `dependencies.json` file indeed.
That is fixed with last release v1.1.1. Cf commit 606654ece8320475e0f1c192e51b25339cfc342f
Thanks for the report !
